### PR TITLE
spiders: ArXiv spider

### PR DESCRIPTION
**Pull request by Lilykos**
* Adds extraction of metadata of arXiv exports.

* Adds documentation and tests.

* Adds documentation for HepRecord item attributes.

Signed-off-by: Ilias Koutsakis <ilias.koutsakis@cern.ch>

### CODE REVIEW

File: hepcrawl/spiders/arxiv_spider.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+# Copyright (C) 2015 CERN.
+#
+# hepcrawl is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Spider for arXiv."""
+from scrapy import Request, Selector
+from scrapy.spiders import XMLFeedSpider
+from ..mappings import CONFERENCE_PAPERS
```
Comment (jalavik) [inline hepcrawl/spiders/arxiv_spider.py]: You can add an empty line here before the import for better style.

File: tests/test_arxiv.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of hepcrawl.
+# Copyright (C) 2015 CERN.
```
Comment (jalavik) [inline tests/test_arxiv.py]: These should be 2016 instead of 2015 all across.

File: hepcrawl/mappings.py [hunk 3]
```diff
@@ -84,3 +84,38 @@
     r'^OA$',
```
Comment (jalavik) [inline hepcrawl/mappings.py]: Add 2016 to the list of copyright years.

File: hepcrawl/items.py [hunk 4]
```diff
@@ -50,14 +50,41 @@ class HEPRecord(scrapy.Item):
         }
     """
```
Comment (jalavik) [inline hepcrawl/items.py]: Add 2016 to the list of copyright years.

File: hepcrawl/spiders/arxiv_spider.py [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+from ..utils import get_first
+
+from ..items import HEPRecord
+from ..loaders import HEPLoader
+
+
+class ArxivSpider(XMLFeedSpider):
+
+    name = 'Arxiv'
```
Comment (jalavik) [inline hepcrawl/spiders/arxiv_spider.py]: Name should be `arXiv` to keep it consistent with most references to this site.

**Comment by Lilykos** [comment]
Updated!

File: hepcrawl/mappings.py [hunk 6]
```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 #
 # This file is part of hepcrawl.
-# Copyright (C) 2015 CERN.
+# Copyright (C) 2016 CERN.
```
Comment (jalavik) [inline hepcrawl/mappings.py]: Should be `2015, 2016`

File: hepcrawl/items.py [hunk 7]
```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 #
 # This file is part of hepcrawl.
-# Copyright (C) 2015 CERN.
+# Copyright (C) 2016 CERN.
```
Comment (jalavik) [inline hepcrawl/items.py]: Should be `2015, 2016`

**Pull request merged**